app/styles/_pipeline.less

```diff
@@ -145,8 +145,11 @@
       text-align: center;
     }
     // hide arrow on last stage
-    &:last-child:before {
-      display: none;
+    &:last-child {
```

Comment:
Do you feel strongly about this change?  I prefer to avoid unnecessary nesting...